tests/functional/meson.build

```diff
@@ -15,6 +15,10 @@ tests = {
         'suites': ['python', 'short'],
         'depends': hse_python_depends,
     },
+    'kvs_many_pfx': {
+        'suites': ['python', 'long'],
```

Comment:
On one of our lab machines with an nvme drive, can you quantify what how long it takes?

Reply:
79 secs on an nvme drive.

Reply:
Does this need to be marked as long then? How long does it take on a github runner (Bhavesh can you a machine to test)? Should this also be marked as `functional`?

Reply:
Yes, functional would be the right suite for this test

Reply:
Here are other tests in the long suite for comparison. My suggestion is to leave it as is.. as this is not the fastest in the suite.

1/3 hse:functional+python+long / kvs_scan_a_million              OK                3.42s
2/3 hse:functional+python+long / kvs_many_pfx                    OK               81.25s
3/3 hse:functional+python+long / kvs_compare_full_and_pfx        OK              118.57s

Reply:
Jacob, do you not want this test to run in the CI?

Reply:
ideally, nightly would be good enough.